packages/elements/src/chart/index.ts

```diff
@@ -609,7 +592,7 @@ export class Chart extends BasicElement {
   protected render (): TemplateResult {
     return html`
       <div container>
```

Comment:
Not sure why we were using `container` attribute to style `div`

Reply:
It specify style in internal template like below.
```
[container] {
        position: absolute;
        top: 0;
        right: 0;
        bottom: 0;
        left: 0;
        display: flex;
        flex-direction: column;
}
```